Helper/Customer.php

```diff
@@ -264,21 +264,32 @@ public function isSubscriberExists($email, $storeId = null)
 
     /**
      * @param string $email
-     * @param int $websiteId
+     * @param \Drip\Connect\Model\Configuration $config
      *
      * @return bool
      */
-    public function isCustomerExists($email, $websiteId = null)
+    public function isCustomerExists($email, \Drip\Connect\Model\Configuration $config)
     {
-        if ($websiteId == null) {
-            $websiteId = $this->storeManager->getStore()->getWebsiteId();
-        }
+        $websiteId = $config->getWebsiteId();
 
         $customer = $this->customerCustomerFactory->create()->setWebsiteId($websiteId)->loadByEmail($email);
 
         return (bool) $customer->getId();
```

Comment:
Same comment as M1 about possibly refactoring this to use `getCustomerByEmail`. The whole method becomes:
```php
return (bool) $this->getCustomerByEmail($email, $config);
```